### PR TITLE
♻️ Upgrade dependencies + support several icons + config mgmt

### PR DESCRIPTION
**Pull request by Resousse**
- Upgrade of dependencies including RobotGo v1
- Adding of 3 more icons for the systray, linked to https://github.com/prashantgupta24/automatic-mouse-mover/issues/41
- Keep icon choice in a config file

**This doesn't work as is, as it's blocked by this PR : https://github.com/prashantgupta24/activity-tracker/pull/28** 
Once the linked PR will be merged and transformed into a new release/tag, the go.mod & go.sum file of this PR would require an update.

### CODE REVIEW

**Comment by Resousse** [comment]
Hello @prashantgupta24 , this PR is not ready to be merged, but this one can be merged (and blocks this one): https://github.com/prashantgupta24/activity-tracker/pull/28

File: cmd/main.go [hunk 1]
```diff
@@ -49,10 +122,17 @@ func onReady() {
...
+			case <-cloud.ClickedCh:
+				setIcon("cloud", configFile)
+			case <-man.ClickedCh:
+				setIcon("man", configFile)
+			case <-geometric.ClickedCh:
+				setIcon("geometric", configFile)
 			case <-about.ClickedCh:
 				log.Infof("Requesting about")
-				robotgo.ShowAlert("Automatic-mouse-mover app v1.2.0", "Developed by Prashant Gupta. \n\nMore info at: https://github.com/prashantgupta24/automatic-mouse-mover")
+				robotgo.Alert("Automatic-mouse-mover app v1.2.0", "Developed by Prashant Gupta. \n\nMore info at: https://github.com/prashantgupta24/automatic-mouse-mover", "OK", "")
```
Comment (prashantgupta24) [inline cmd/main.go:135]:
```suggestion
				robotgo.Alert("Automatic-mouse-mover app v1.3.0", "Developed by Prashant Gupta. \n\nMore info at: https://github.com/prashantgupta24/automatic-mouse-mover", "OK", "")
```

**Comment by prashantgupta24** [comment]
@Resousse thanks so much for this PR!

**Comment by prashantgupta24** [comment]
@Resousse feel free to update the `go.sum` file with the latest changes

**Review by prashantgupta24 — APPROVED** [review]
Looks pretty good to me!

**Pull request merged**